tensorflow/lite/toco/graph_transformations/unpartition_embedding_lookup.cc

```diff
@@ -57,10 +57,10 @@ ::tensorflow::Status UnpartitionEmbeddingLookup::Run(Model* model,
   // Split up the DynamicStitch inputs into the indices and data.
   std::vector<string> stitch_indices_inputs;
   std::vector<string> stitch_data_inputs;
-  for (size_t i = 0; i < stitch_op->num_partitions; ++i) {
+  for (int i = 0; i < stitch_op->num_partitions; ++i) {
     stitch_indices_inputs.push_back(stitch_op->inputs[i]);
   }
-  for (size_t i = stitch_op->num_partitions; i < stitch_op->num_partitions * 2;
+  for (int i = stitch_op->num_partitions; i < stitch_op->num_partitions * 2;
```

Comment:
What's the reason for going from size_t to int here, whereas this commit goes from int to size_t elsewhere?